packages/block-editor/src/components/block-list/use-block-props/use-firefox-draggable-compatibility.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * WordPress dependencies
+ */
+import { useRefEffect } from '@wordpress/compose';
+
+export function useFirefoxDraggableCompatibility() {
```

Comment:
Can you add a comment explaining what this fixes?

Reply:
Added in 76f61df275 and rebased this PR too to get the latest drag and drop fixes.